lib/sinon/match.js

```diff
@@ -8,7 +8,6 @@
  */
 "use strict";
 
-var create = require("./util/core/create");
 var deepEqual = require("./util/core/deep-equal").use(match); // eslint-disable-line no-use-before-define
 var every = require("./util/core/every");
```

Comment:
It'll require a bit of refactoring of the code, as the `every` function is designed to also work with `Object`, because someone decided that they are "iterables"....